exporter/clickhouselogsexporter/exporter.go

```diff
@@ -249,6 +249,17 @@ func (e *clickhouseLogsExporter) pushToClickhouse(ctx context.Context, ld plog.L
 			resources = addTemporaryUnderscoreSupport(resources)
 
 			for j := 0; j < logs.ScopeLogs().Len(); j++ {
+				scope := logs.ScopeLogs().At(j).Scope()
+				scopeName := scope.Name()
+				scopeVersion := scope.Version()
+
+				scopeAttributes := attributesToSlice(scope.Attributes(), true)
+
+				err := addAttrsToTagStatement(tagStatement, "instrumentation_scope", resources)
```

Comment:
This should be using `scopeAttributes`

Reply:
Have updated it to `scope` https://github.com/SigNoz/signoz-otel-collector/pull/344